lib/python/picongpu/plugins/plot_mpl/requirements.txt

```diff
@@ -0,0 +1,2 @@
+matplotlib
+numpy
```

Comment:
pls include this file in the `requirements.txt` a level above

Reply:
Sorry, I'm not sure how to write that correctly, do you mean `plugins/requirements.txt` would then look like this?
```
numpy
pandas>=0.21.0
h5py
matplotlib
-r .plot_mpl/requirements.txt
```

Reply:
Close, just without the dot:

```bash
$ cat requirements.txt
-r lib/python/picongpu/requirements.txt

$ cat lib/python/picongpu/requirements.txt
-r input/requirements.txt
-r plugins/requirements.txt
-r utils/requirements.txt

$ cat lib/python/picongpu/plugins/requirements.txt
numpy
pandas>=0.21.0
h5py
matplotlib 
-r plot_mpl/requirements.txt
```

Reply:
Thanks, added this line now.